python/models.md

```diff
@@ -0,0 +1,9 @@
+# Model style
+
+```
+title = models.CharField(
+    max_length=100,
+    blank=True,
+    null=True,
```

Comment:
Yup, I'm in agreement with LC. Dangling comma in Python, but not in JS is normal.